node/forcedroid.js

```diff
@@ -38,7 +38,8 @@ var path = require('path');
 var outputColors = require('../external/shared/node/outputColors');
 var commandLineUtils = require('../external/shared/node/commandLineUtils');
 var shelljs = require('shelljs') ;
-var fs = require('fs'); 
+var fs = require('fs');
```

Comment:
We chose ```fs``` because it comes preinstalled with ```node```. Is that the case for ```fs-extra``` too? I fear that we'll run into the same issues as ```shelljs``` by introducing an extra dependency.